library/include/df/custom/coord2d.methods.inc

```diff
@@ -1,6 +1,6 @@
 coord2d(uint16_t _x, uint16_t _y) : x(_x), y(_y) {}
 
-bool isValid() const { return x != -30000; }
+bool isValid() const { return x >= 0; }
```

Comment:
This can definitely have some false _positives_, since there is no guarantee that an uninitialized integer is negative. A large positive number could also easily be out of map bounds.

Reply:
That's true, but for the purposes here, lack of a cursor should still be in the neighborhood of -30000, we just don't know the exact value anymore

Reply:
Ok, I found your post on Discord. I don't have a link handy, but I'd summarize it as: the cursor does seem to be initialized to -30000, but then incremented/decremented by at most some multiple of the map size, so it should always remain negative (probably within 1,000 of -30,000, even).

So I'm ok with this, although it's not ideal.